.github/workflows/upload-perf-results.yml

```diff
@@ -93,3 +93,19 @@ jobs:
         export PGPORT=$(cat ${{github.workspace}}/PGPORT)
         export PGDATABASE=$(cat ${{github.workspace}}/PGDATABASE)
         psql -f ${{github.workspace}}/results/upload.sql
+
+    - name: Download and run regression script
+      run: |
+        curl https://raw.githubusercontent.com/microsoft/bpf_performance/refs/heads/main/scripts/check_perf_results.sql > check_perf_results.sql
```

Comment:
Do we need to download the script from the internet.? Since the source code can be checked, can't we access the sql script like scripts/check_perf_results.sql

Reply:
It's from a different repo.